api/generate.go

```diff
@@ -47,6 +48,7 @@ func main() {
 				RenderClients:    true,
 				RenderTypes:      true,
 				ApiRoot:          "pkg/api",
+				CustomTemplates:  contrib.AllCustomTemplates,
```

Comment:
i'm not sure we want `AllCustomTemplates` here, since we'll wind up with unused code as things get added to `contrib`